examples/k8s_sa.py

```diff
@@ -0,0 +1,45 @@
+"""This example showcases the hello world example of Hera using k8s service account token"""
+from hera.v1.task import Task
+from hera.v1.workflow import Workflow
+from hera.v1.workflow_service import WorkflowService
+
+from kubernetes import client, config
+import base64
+
+
+def get_sa_token(service_account: str, namespace: str = "default", config_file=None):
+    """Get ServiceAccount token using kubernetes config.
+
+     Parameters
+    ----------
+    service_account: str
+        The service account to authenticate from.
+    namespace: str = 'default'
+        The K8S namespace the workflow service submits workflows to. This defaults to the `default` namespace.
+    config_file: str
+        The path to k8s configuration file.
+    """
+    config.load_kube_config(config_file=config_file)
+    v1 = client.CoreV1Api()
+    secret_name = (
+        v1.read_namespaced_service_account(service_account, namespace).secrets[0].name
+    )
+    sec = v1.read_namespaced_secret(secret_name, namespace).data
+    return base64.b64decode(sec["token"]).decode()
+
+
+namespace = "argo"
+token = get_sa_token("argo-server", namespace=namespace)
```

Comment:
Any chance we can position these variables right before the `WorkflowService` instantiation? It might help keep everything relevant for the workflow submission example neatly in the same block 🙂

Reply:
done :slightly_smiling_face: